TestProjects/NotificationSamples_2018/Assets/Scripts/AndroidManualTests.cs

```diff
@@ -0,0 +1,328 @@
+#if PLATFORM_ANDROID
```

Comment:
Isn't this a duplicate of TestProjects/NotificationSamples/Assets/Scripts/AndroidManualTests.cs

If so, why not put it in com.unity.mobile.notifications/com.unity.mobile.notifications/Tests/ folder?

Or use some other trick, like create a local package for file sharing

Reply:
yeah... I agreed. Will find a better way to do this.